src/helpers/redux/sagas/auth.ts

```diff
@@ -0,0 +1,59 @@
+import { SET_USER_AUTH_REQUEST } from 'helpers/constants';
+import {
+    addUserAuthToStateSuccess,
+    addUserAuthToStateFailure,
+} from 'helpers/redux/actions/auth';
+import { IUserAuth } from 'helpers/types/endpoints';
+import * as Sentry from 'sentry-expo';
+import Api from 'services/api';
+import { call, put, all, takeEvery } from 'typed-redux-saga';
+
+const getUser = async (
```

Comment:
This was just a way to to decouple code dependency and better readability to the code. Although this SAGA isn't in use because we used promise instead in this user case.